x-pack/test/functional/page_objects/monitoring_page.js

```diff
@@ -23,19 +22,9 @@ export function MonitoringPageProvider({ getPageObjects, getService }) {
     }
 
     async assertTableNoData(subj) {
-      await retry.try(async () => {
-        if (!await testSubjects.exists(subj)) {
-          throw new Error('Expected to find the no data message');
-        }
-      });
-    }
-
-    async assertEuiTableNoData(subj) {
-      await retry.try(async () => {
-        if (await testSubjects.exists(subj)) {
-          throw new Error('Expected to find the no data message');
-        }
-      });
+      if (!await testSubjects.exists(subj)) {
```

Comment:
This logic should match the original table design - we check for the existence of the `noData` selector